pkg/apply/duration/duration.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package duration
+
+import "time"
+
+var startTime time.Time
```

Comment:
This would make it impossible to use this package (and all the packages that depend on it) concurrently. Why not just use a local variable?